.travis.yml

```diff
@@ -1,4 +1,5 @@
 os:
+  - linux
```

Comment:
This line was present in the original travis.yml file in the bioconda repo. I removed it in an earlier commit and reverted it back later to the original one.